src/v/debug_bundle/json.h

```diff
@@ -105,7 +105,8 @@ debug_bundle::result<T> from_json(const json::Value& v) {
         if (v.IsString()) {
             // TODO: Improve this with HowardHinnant/date or
             // std::chrono::zoned_time
-            std::istringstream ss(v.GetString(), v.GetStringLength());
+            std::string ss_str(v.GetString(), v.GetStringLength());
+            std::istringstream ss(std::move(ss_str));
```

Comment:
>How strange, which constructor was it even using?

Right? I don't see anything listed here which looks like it would work, right?

https://en.cppreference.com/w/cpp/io/basic_istringstream/basic_istringstream